### DISCUSSION TITLE
Modelling ducted fans

### DISCUSSION THREAD

**Discussion by agrewalks** [byuflowlab/FLOWUnsteady#75] [General]
Congratulations on this very versatile and powerful framework!  Can the Actuator Surface Model be used to define a duct around a fan?
Thanks

**Comment by CFD-designs** [comment]
There is some mention of a ducted fan example in the instructions, but I couldn't see the code for it anywhere or how it was generated.

**Comment by EdoAlvarezR** [comment]
Hi Anant! 

I did a proof of concept simulating a shrouded rotor with the wing actuator surface model (ASM) in this simulation: [LINK](https://www.youtube.com/watch?v=Bf4pIt7oi5k). However, I didn't feel like this was the appropriate model to capture the effects of a surface in very close proximity to the blade tips, so I moved forward doing validation (perhaps the wing ASM could still be used to roughly capture some effects).

Instead, we developed a new ASM using a panel method which we showed in the AVIATION 2023 paper: [[Video]](https://www.youtube.com/watch?v=ebOW4aLGlMA) [[Paper]](https://scholarsarchive.byu.edu/cgi/viewcontent.cgi?article=7676&context=facpub). This model works well for ducted fans, but more development and validation is still underway. That work is currently in the [flowpanel_prototype branch](https://github.com/byuflowlab/FLOWUnsteady/tree/flowpanel-prototype). If you are interested in helping the development and validation process, we could send you the scripts that run the simulations shown in the paper for you to start running your own simulations.

**Reply by CFD-designs** [reply]
The paper is pretty cool. 
The issue though is that this approach requires manual specification of the separation points, which is why the upper body is not giving accurate results compared to the lower body.
With regard to rotor gap, is there any scaling used or adaptive mesh like some LBM solvers?

**Reply by EdoAlvarezR** [reply]
I think you might be right.

Regarding the blade tip gap, the gap is directly defined by the geometry that the user gives (it's simply the spacing between the rotor diameter and the duct inner diameter at that station). However, the duct is inviscid and there's still a lot of validation to do to understand the limitations of this type of model.

**Reply by ilm71** [reply]
Hi Ed, 

would you mind to send me the scripts mentioned for the simulation of the ducted fans? I am currently trying to simulate rotor-structure-interactions roughly with FLOWUnsteady to cover the aerodynamic effects. I have already adjusted my code by sticking to the shown scripts in the flow-panel prototype branch, but in my opinion there are currently issues in terms of that the rotor downwash still penetrates the structure volume. 
Thanks!

**Reply by EdoAlvarezR** [reply]
Related to #144

**Reply by Burning1997** [reply]
Hi Ed,

Would you be able to share the scripts you mentioned for the simulation of ducted fans? I’m currently working on simulating rotor-structure interactions, particularly using FLOWUnsteady to capture the aerodynamic effects.

Thanks so much in advance!

**Comment by kont1901** [comment]
Hi Alvarez,
Thanks for your discussions on ducted rotor. I am also interested in these models described in your paper ( AVIATION 2023 paper) as Dr. Anant mentioned. It is a very good paper. Is it possible to share your models on the duct and on the duct-rotor coupling?
Thanks
Charly

**Comment by agrewalks** [comment]
Hi Ed,
My colleague Tenon Charly Kone and I would like to help with the development and validation of a ducted rotor model. You had mentioned that would be willing to share the scripts used to run the ducted rotor sims in your 2023 AIAA Aviation paper “Vortex Particle Method for Electric Ducted Fan in Non-Axisymmetric Flow.”  That would be a great place to start.  We will familiarize ourselves with the script ad we can follow up with a virtual meeting.
Regards
Anant

**Comment by EdoAlvarezR** [comment]
@kont1901 @agrewalks, it would be great if you guys could help validating the ducted fan model. I believe there's great potential using panels for the duct and actuator line for the rotor as shown in the paper (unfortunately, I have been busy with other things at work after the AVIATION 2023 paper). 

@cibinjoseph recently picked up the ducted fan model to continue working towards a journal paper. Perhaps we can do a collaboration jumping together in the same paper.

I'll send you my email over LinkedIn to continue the conversation through email.
